src/java.base/share/classes/sun/security/util/ECParameters.java

```diff
@@ -23,7 +23,7 @@
  * questions.
  */
 
-package sun.security.util;
+package java.security;
```

Comment:
Hello @johnshajiang, was this change to the package name intentional? Looking at the description in the JBS issue, this looks like an accidental change, isn't it?

Reply:
My fault!
Already submitted another JBS for this issue
https://bugs.openjdk.java.net/browse/JDK-8282166